tests/test_timing_properties.py

```diff
@@ -0,0 +1,143 @@
+"Contains a collection of pytest tests that validates the timing properties."
+
+import random
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx import DaqError
+from nidaqmx.constants import AcquisitionType, SampleTimingType
+
+
+class TestTimingProperty(object):
+    """Contains a collection of pytest tests that validates the timing properties."""
+
+    def test_boolean_property(self, any_x_series_device):
```

Comment:
NI's regression testing conventions: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions

Rename using given/when/then syntax. This may require splitting the test cases.